### PR TITLE
#184581202-User should be prompted to change their passwords every x amount of time

### PR DESCRIPTION
**Pull request by wilbrord2**
#### What does this PR do?
This PR if for  prompting Users to change their passwords every x amount of time since the time it was either manually updated.

#### Description of Task to be completed?
- setting deadline for password
-  developing functionality for checking if user password is expired
- User should be able to update his/her password

#### How should this be manually tested?
- Clone this repo git clone https://github.com/atlp-rwanda/e-comm-team-amigos-bn
- Switch your working directory cd e-comm-team-amigos-bn
- Create an .env file and update it with .env-example
- Install all the dependencies by running npm install
- Then run the project by typing npm start

#### Endpoints to be tested:
- `user/login`
- `user/createPassword`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker/Trello stories?

Story ID: [#184581202(https://www.pivotaltracker.com/story/show/184581202)

#### Screenshots (if appropriate)
Screenshot (164)
Screenshot (163)
Screenshot (172)

### CODE REVIEW

**Review by Rwibutso — APPROVED** [review]

**Review by GasanaJr — CHANGES_REQUESTED** [review]
rebase on develop, otherwise it is good to go

**Review by GasanaJr — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -14,8 +14,8 @@ COVERALLS_REPO_TOKEN=covernal_token
 DATABASE_URL_PROD=production_url
 
 #
-AUTH_USER=Use_your_current_email
-AUTH_PASS=User_your_2-step-verification_Password_on_your_email
+AUTH_USER=amigosteam70@gmail.com
+AUTH_PASS=qauhadtdxpkwwdmk
```
Comment (Habinezajanvier) [inline .env.example]: Please put those value in `.env` only and remain with their placeholders in `.env.example`

**Review by Habinezajanvier — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (wilbrord2) [reply .env.example]: All changes are added

**Review by AbdulKhaliq59 — APPROVED** [review]

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**